features/full_tests/batch_1/detect_anr_jvm.feature

```diff
@@ -14,25 +14,6 @@ Scenario: ANR triggered in JVM loop code is captured
     And the error payload field "events.0.threads.0.type" equals "android"
     And the error payload field "events.0.threads.0.stacktrace.0.type" is null
 
-# Other scenarios use a deadlock to generate an ANR, which works on Samsung devices. This scenario remains skipped
```

Comment:
Note: This is being removed because the new detection code can no longer tell the difference between Java functions that syscall, and actual native functions.